requirements.txt

```
@@ -1,2 +1,2 @@
-karton.core==4.0.5
+karton-core==4.0.5
```

```suggestion
karton-core==4.2.0
```

We might as well bump it